packages/building-rollup/default-config.js

```diff
@@ -0,0 +1,73 @@
+import resolve from 'rollup-plugin-node-resolve';
+import { terser } from 'rollup-plugin-terser';
+import babel from 'rollup-plugin-babel';
+import indexHTML from './plugins/rollup-plugin-index-html';
+
+const production = !process.env.ROLLUP_WATCH;
+const prefix = '[owc-building-rollup]';
+
+function createConfig(options, legacy) {
+  if (!options.input) {
+    throw new Error(`${prefix}: missing option 'input'.`);
+  }
+
+  if (typeof options.input !== 'string' || !options.input.endsWith('.html')) {
+    throw new Error(`${prefix}: input should point to a single .html file.`);
+  }
+
+  return {
+    input: options.input,
+    output: {
+      // output into given folder or default /dist. Output legacy into a /legacy subfolder
+      dir: `${options.output || 'dist'}${legacy ? '/legacy' : ''}`,
+      // we can't reliably serve esm output, as we can't detect dynamic import support at the moment
+      format: legacy ? 'system' : 'esm',
+      sourcemap: true,
+    },
+    plugins: [
+      indexHTML({
+        legacy,
+        polyfillWebComponents: legacy,
+      }),
+      // resolve bare import specifiers
+      // TODO: Check options
+      resolve(),
+      babel({
```

Comment:
There's really no reason to run babel on a modern build. babel should be disabled in favour of terser for modern browsers, imo.

Reply:
That's debatable. If you want to use new syntax which isn't yet supported on all evergreens you still need babel to handle that for you. So it's question of what is the best default here.

not sure what babel has to do with terser though?

Reply:
https://open-wc.org/about/rationales.html#rationales
> Language or platform features/APIs must be released without a flag in the stable version of a least one browser
> Note: We currently have ONE exception to this rule and that is 'bare modules'.

We can show how to add their own babel config, but 'babel all the things' as default for modern browsers is a mistake, in my opinion.

Reply:
If we don't 'babel all the things', it means that people won't be able to use new javascript features until they are supported by chrome, safari, firefox and edge. If only one browser supports it, you still can't ship it to production.

Reply:
Right, I'm advocating that we recommend against using features that aren't widely available.

I also advocate that we show examples for how to opt in to babel transforms for when you do want to use features like private fields or decorators.

Reply:
This is, of course, a matter of opinion, and i'm happy to defer to the group if it's decided that babel is always necessary..

but in my experience it's not, and babel transforms aren't free - they often add lots of overhead in terms of bundle size and runtime requirements.

Reply:
I understand what you're saying. I can go either way here, personally I prefer to use new syntax when it becomes part of the spec.

Transform size is a problem for older specs, like regenerator. I don't see this with newer specs but that that might come again with future specs. 

The question is what's the best default here.

Reply:
we actually do need babel for the import meta transform, because you need the URLs from before the bundling rather than after